data/json/vehicles/test.json.rej

```diff
@@ -0,0 +1,96 @@
+--- data/json/vehicles/test.json
```

Comment:
Is this file added by accident?

Reply:
Possibly, this is one of the files in the original commits by the looks of it. I'll get it out of there if it's not needed for anything.